deploy/operator.yaml

```diff
@@ -31,7 +31,9 @@ spec:
           imagePullPolicy: Always
           env:
             - name: WATCH_NAMESPACE
-              value: ""
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
These changes belong to a different PR, why are they showing up here?

Reply:
I had to merge in the changes from the master branch; this comes from the merge commit:

commit 1a01d68ef14e46f41fc131934cf3900cfc04c784 (HEAD -> master-with-lease-support, myform/master-with-lease-support)
Author: Michael Moser <moser.michael@gmail.com>
Date:   Thu May 28 05:55:32 2020 +0300

    fix WATCH_NAMESPACE env variable in container spec of generated CSV (#82)
    
    fixing the operator deployment object was not good enough.
    generate csv seems to be taking it from some internal template.
    
    Signed-off-by: Michael Moser <mmoser@redhat.com>
    
    Co-authored-by: Michael Moser <mmoser@redhat.com>

```